airavata-courses/scrapbook
# Authentication Service

The service decodes the JWT token to get the issuer. Depending on the issuer, it verifies the token with the clientID.

- **Port**: 5000
- **API**: `/login?token={token}`
- **Response**
   - **200**: if token valid 
   - **401**: if token invalid

Installation guide given in the service README

### How to test?
- Log in using the frontend
- Go to the Google Developer Console -> Application -> local storage -> http://localhost:4200 -> grab the `scrapbook-token` and paste it in a text field.
- Start the auth-service
- make a request to `http://localhost:5000/auth?token={your_token}` 
- See the response on the broswer/postman
